### PR TITLE
Update codeflare-sdk version on imagestreams annotations

### PR DESCRIPTION
**Pull request by atheo89**
Follow up PR to this: https://github.com/opendatahub-io/notebooks/pull/515
related to:  https://issues.redhat.com/browse/RHOAIENG-6271

### CODE REVIEW

File: manifests/base/jupyter-tensorflow-notebook-imagestream.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ spec:
...
+        opendatahub.io/notebook-python-dependencies: '[{"name":"TensorFlow","version":"2.15"},{"name":"Tensorboard","version":"2.15"},{"name":"Boto3","version":"1.34"},{"name":"Kafka-Python","version":"2.0"},{"name":"Kfp","version":"2.7"},{"name":"Matplotlib","version":"3.8"},{"name":"Numpy","version":"1.26"},{"name":"Pandas","version":"2.2"},{"name":"Scikit-learn","version":"1.4"},{"name":"Scipy","version":"1.12"},{"name":"Odh-Elyra","version":"3.16"},{"name":"PyMongo","version":"4.6"},{"name":"Pyodbc","version":"5.1"}, {"name":"Codeflare-SDK","version":"0.16"}, {"name":"Sklearn-onnx","version":"1.16"}, {"name":"Psycopg","version":"3.1"}, {"name":"MySQL Connector/Python","version":"8.3"}]'
```
Comment (jstourac) [inline manifests/base/jupyter-tensorflow-notebook-imagestream.yaml:21]: Shouldn't this be Kfp 2.5? :thinking: 

```suggestion
        opendatahub.io/notebook-python-dependencies: '[{"name":"TensorFlow","version":"2.15"},{"name":"Tensorboard","version":"2.15"},{"name":"Boto3","version":"1.34"},{"name":"Kafka-Python","version":"2.0"},{"name":"Kfp","version":"2.5"},{"name":"Matplotlib","version":"3.8"},{"name":"Numpy","version":"1.26"},{"name":"Pandas","version":"2.2"},{"name":"Scikit-learn","version":"1.4"},{"name":"Scipy","version":"1.12"},{"name":"Odh-Elyra","version":"3.16"},{"name":"PyMongo","version":"4.6"},{"name":"Pyodbc","version":"5.1"}, {"name":"Codeflare-SDK","version":"0.16"}, {"name":"Sklearn-onnx","version":"1.16"}, {"name":"Psycopg","version":"3.1"}, {"name":"MySQL Connector/Python","version":"8.3"}]'
```

Comment (harshad16) [reply manifests/base/jupyter-tensorflow-notebook-imagestream.yaml:21]: good catch, as we had made the change directly in 2.9 
thanks

**Review by harshad16 — APPROVED** [review]
/lgtm
/approve

thanks :100:

**Comment by harshad16** [comment]
/lgtm
with all the changes.
merging it

**Pull request merged**